index.css

```
@@ -0,0 +1,330 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    overflow-x: hidden;
+}
+a {color: inherit; text-decoration: none; outline: none} a:hover, a:active {text-decoration: none;}
```

❗️ 수정요청
한 파일내에서는 같은 룰을 공유하는 것이 좋습니다. 또한 적절한 개행은 가독성에 도움이됩니다. 이 부분도 다른 부분처럼 개행하시는 것을 추천드려요~

```suggestion
* {
    margin: 0;
    padding: 0;
    box-sizing: border-box;
    overflow-x: hidden;
}

a {
    color: inherit; text-decoration: none; outline: none
} 

a:hover, a:active {
    text-decoration: none;
}
```